### DISCUSSION TITLE
Make Inbox text clickable

### DISCUSSION THREAD

**Discussion by abctje** [jonaswinkler/paperless-ng#897] [Feature Requests]
A small but nice little change would be a if the Inbox text is clickable and takes you directly to the inbox ...I try to do everything to avoid extra mouse clicks ;-)

I've pointed it out in the image. The text is is my native language.
![screenshot](https://user-images.githubusercontent.com/36775538/114358702-7c118780-9b73-11eb-9fa1-ee4a2cbc63c4.png)

**Comment by colinfrei** [comment]
Fully agree, I think this every time I see that number is bigger than 0, nice to see there's already a feature request for it :)
